packages/devkit/src/utils/package-json.ts

```diff
@@ -287,15 +287,18 @@ export function removeDependenciesFromPackageJson(
     )
   ) {
     updateJson(tree, packageJsonPath, (json) => {
-      for (const dep of dependencies) {
-        delete json.dependencies[dep];
+      if (json.dependencies) {
+        for (const dep of dependencies) {
+          delete json.dependencies[dep];
+        }
+        json.dependencies = sortObjectByKeys(json.dependencies);
```

Comment:
Since we are improving this code if there are no dependencies we could also remove the dependencies instead of attempting to sort it.

Reply:
I think we can do this separately.